pyethapp/accounts.py

```diff
@@ -480,9 +480,7 @@ def get_by_address(self, address):
         assert len(address) == 20
         accounts = [account for account in self.accounts if account.address == address]
         if len(accounts) == 0:
-            raise KeyError('account not found by address', address=address.encode('hex'))
-        elif len(accounts) > 1:
-            log.warning('multiple accounts with same address found', address=address.encode('hex'))
```

Comment:
My bad, I thought it was dead code because I thought it was prevented upfront here:
https://github.com/ethereum/pyethapp/blob/v1.5.0/pyethapp/accounts.py#L283
Unit test (test_add_account_twice):
https://github.com/ethereum/pyethapp/blob/v1.5.0/pyethapp/tests/test_account_service.py#L104
But that's only on known uuid, so let me update the PR.